janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -1221,10 +1246,15 @@ private void addOrderToQuery(KeyInformation.IndexRetriever informations, Elastic
     @Override
     public Long totals(RawQuery query, KeyInformation.IndexRetriever information,
                        BaseTransaction tx) throws BackendException {
-        final int size = query.hasLimit() ? Math.min(query.getLimit() + query.getOffset(), batchSize) : batchSize;
-        final ElasticSearchResponse response = runCommonQuery(query, information, tx, size, false);
-        log.debug("Executed query [{}] in {} ms", query.getQuery(), response.getTook());
-        return response.getTotal();
+        long count;
+        if(log.isDebugEnabled()){
```

Comment:
I think such a implementation is critical, because someone can forget to update one branch.

Reply:
@farodin91 I am not sure I understand your question.
Before this PR we was returning a time spent in the search query and were returning that time if debug is enabled. Now we are not counting the time in the count query. We are counting the time spent to process the query if debug is enabled. Otherwise, we don't count the time.

I am just not sure about what branch you are speaking and what can we forget to update?

Reply:
You call in both if branches runCountQuery, if someone change it, both lines have to be changes which is a bit annoying, because you can forget this easily.

Reply:
You are right. I have changed it to:
```
long startTime = System.currentTimeMillis();
long count = runCountQuery(query);
if(log.isDebugEnabled()){
    log.debug("Executed count query [{}] in {} ms", query.getQuery(), System.currentTimeMillis() - startTime);
}
return count;
```